core/state-machine/src/lib.rs

```diff
@@ -245,7 +253,13 @@ pub trait Externalities<H: Hasher> {
 
 	/// Get the change trie root of the current storage overlay at a block with given parent.
 	fn storage_changes_root(&mut self, parent: H::Out) -> Result<Option<H::Out>, ()> where H::Out: Ord;
+}
 
+/// Extra externalities API: pinned to specific active address.
```

Comment:
What active address? I bet that was from super old times when there were contracts in substrate/polkadot. I.e. totally irrelevant